include/NovelRT.Interop/NrtDebugService.h

```diff
@@ -4,23 +4,20 @@
 #ifndef NOVELRT_INTEROP_DEBUGSERVICE_H
 #define NOVELRT_INTEROP_DEBUGSERVICE_H
 
-#include "Graphics/NrtRenderingService.h"
-#include "Utilities/NrtCommonEvents.h"
```

Comment:
More missing includes. See previous comment. I'll stop commenting on them after this one if I see any others.